cms/templates/settings.html

```diff
@@ -292,6 +292,26 @@ <h2 class="title-2">${_("Introducing Your Course")}</h2>
                 <span class="tip">${_("Information for prospective students")}</span>
               </header>
               <ol class="list-input">
+
+                % if about_page_editable:
+                <li class="field" id="field-course-title">
+                  <label for="course-title">${_("Course Title")}</label>
+                  <input type="text" id="course-title" data-display-name="${context_course.display_name}">
+                </li>
+                <li class="field" id="field-course-subtitle">
+                  <label for="course-subtitle">${_("Course Subtitle")}</label>
```

Comment:
Nit: You can nest the input within the label element, removing the need to link them together with "for".